device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClientConfig.java

```diff
@@ -31,6 +31,9 @@
     private static final int DEFAULT_HTTPS_READ_TIMEOUT_MILLIS = 240000;
     private static final int DEFAULT_HTTPS_CONNECT_TIMEOUT_MILLIS = 0; //no connect timeout
 
+    private static final int DEFAULT_AMQP_OPEN_AUTHENTICATION_SESSION_TIMEOUT = 20; // 20 second timeout
+    private static final int DEFAULT_AMQP_OPEN_DEVICE_SESSIONS_TIMEOUT = 60; // 60 second timeout
```

Comment:
Include a postfix of "_SECONDS" to these two constants to really drive home that they are in terms of seconds

Reply:
good idea. The name was too long so I was on the fence but it would be better.